src/router/Router.js

```diff
@@ -164,6 +169,13 @@ export default class Router extends DisposableBase {
             return this._observeEventsUsingConventions(modelId, object, methodPrefix);
         }
     }
+    addOnErrorHandler(handler) {
+        this._onErrorHandler = handler;
```

Comment:
This method name would imply there is a collection of error handlers, however the functionality actually replaces the last or default registered one.

I think the name is correct, however the underlying store should be an array of handlers.